packages/kit/src/views/Market/components/TokenPriceChart.tsx

```
@@ -237,12 +238,26 @@ function BasicTokenPriceChart({
+}: Omit<IChartProps, 'height'> & {
+  fallbackToChart: boolean;
+  tvPlatform?: IMarketTokenDetail['tvPlatform'];
+}) {
   const [showLoading, changeShowLoading] = useState(true);
   const onLoadEnd = useCallback(() => {
     changeShowLoading(false);
   }, []);
   const ticker = useMemo(() => {
+    if (
+      tvPlatform &&
+      tvPlatform.baseToken &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)